cli/silence_query.go

```diff
@@ -84,6 +85,7 @@ func configureSilenceQueryCmd(cc *kingpin.CmdClause) {
 
 	queryCmd.Flag("expired", "Show expired silences instead of active").BoolVar(&c.expired)
 	queryCmd.Flag("quiet", "Only show silence ids").Short('q').BoolVar(&c.quiet)
+	queryCmd.Flag("created-by", "Only show silences created by this author").StringVar(&c.createdBy)
```

Comment:
```suggestion
	queryCmd.Flag("created-by", "Show silences that belong to this creator").StringVar(&c.createdBy)
```

This is more consistent with the other flag:      ` --within=WITHIN          Show silences that will expire or have expired within a duration`

We use Creator in the UI and code, not author.

Reply:
Yeah, I was a bit unsure about this, given that `amtool silence add` uses `--author`. However, I agree that "creator" is more consistent.